### PR TITLE
[pfcwd]: Add support to poll queue attr in syncd

### PR DESCRIPTION
**Pull request by sihuihan88**

### CODE REVIEW

**Comment by sihuihan88** [comment]
Depends on https://github.com/Azure/sonic-swss-common/pull/141

File: syncd/syncd_pfc_watchdog.cpp [hunk 1]
```diff
@@ -64,6 +71,30 @@ void PfcWatchdog::setQueueCounterList(
...
+
+    auto it = wd.m_queueAttrIdsMap.find(queueVid);
+    if (it != wd.m_queueAttrIdsMap.end())
+    {
+        (*it).second->queueAttrIds = attrIds;
+        return;
+    }
+
+    auto queueAttrIds = std::make_shared<QueueAttrIds>(queueId, attrIds);
+    wd.m_queueAttrIdsMap.emplace(queueVid, queueAttrIds);
```
Comment (marian-pritsak) [inline syncd/syncd_pfc_watchdog.cpp]: Need to erase IDs on delete

Comment (sihuihan88) [reply syncd/syncd_pfc_watchdog.cpp]: Good catch. Updated

File: syncd/syncd_pfc_watchdog.cpp [hunk 2]
```diff
@@ -176,7 +216,6 @@ void PfcWatchdog::collectCounters(
 {
     SWSS_LOG_ENTER();
 
-    std::lock_guard<std::mutex> lock(g_mutex);
```
Comment (stcheng) [inline syncd/syncd_pfc_watchdog.cpp]: remove one more empty line

**Pull request merged**